GETTING_STARTED.md

```diff
@@ -40,7 +40,15 @@ Fetch the latest version of `master` from `upstream` (ie. the main repo):
 
 ### Get it running
 
-When ready, run `script/setup`. If the script succeeds you're ready to start developing. If not, take a look at the output as it should be informative enough to help you troubleshoot.
+First, you need to create the database user the app will use by manually typing the following in your terminal:
+
+```sh
+$ sudo -u postgres psql -c "CREATE USER ofn WITH SUPERUSER CREATEDB PASSWORD 'f00d'"
+```
+
+This will create the "ofn" user as superuser and allowing it to create databases. Note the command will prompt for the password twice. Use "f00d".
```

Comment:
Will it still prompt for the password twice?

Reply:
Oops, nope.